src/routes/index.js

```diff
@@ -1,7 +1,6 @@
-import express from 'express';
 
-const router = express.Router();
+import usersRoute from './api/users';
 
-router.use('/api', require('./api'));
```

Comment:
i don't think you needed to remove this line.... @Oliver-ke , we need express router here...

Reply:
@oyewoas it can be modified with time, i think he  just went plainly with **_'api/v1/users'_**

Reply:
> @oyewoas it can be modified with time, i think he just went plainly with **_'api/v1/users'_**

@funmi5 i get you, i understand what he did, I think he could have still gotten the same result without modifying that file since that convention will be used for endpoints creation.

Reply:
@oyewoas , bro this was really needed.. The file structure became complicated, please take a second look at it